### ISSUE TITLE
Ruby version is broken

### ISSUE THREAD

**Issue by telmofcosta** [Bash-it/bash-it#1099]
Pull Request https://github.com/Bash-it/bash-it/pull/1095 that is now merged broke my theme.

My theme code:
```
...  ${yellow}$(ruby_version_prompt) ...
```
My prompt:
![screen shot 2017-11-27 at 17 46 53](https://user-images.githubusercontent.com/597332/33280743-01c55afe-d39b-11e7-8ae8-da072f1eb532.png)

It is joining `$(rbenv_version_prompt)` with `$(rb_interp_prompt)`.

I think this affects everyone that uses some ruby version manager covered by `bash-it` here
https://github.com/Bash-it/bash-it/blob/master/themes/base.theme.bash#L392

**Comment by telmofcosta** [comment]
One option could be something like this:
```sh
function ruby_version_prompt {
  version=$(rbfu_version_prompt)
  version=${version:-$(rbenv_version_prompt)}
  version=${version:-$(rvm_version_prompt)}
  version=${version:-$(rb_interp_prompt)}
  echo -e "$version"
}
```
Not sure if it's possible to replicate this in a more concise way.

**Comment by lfelipe1501** [comment]
Hi @telmofcosta You could make this pull request to fix this problem, I think it's a good idea what do you propose. 👍

**Comment by LanikSJ** [comment]
I'm seeing the same thing actually.
`100% 2017-11-27 11:20:26 ⌚ |ruby-2.4.2|2.4.2p198 hostname in ~`

**Comment by nwinkler** [comment]
The other option would be to make this optional. Maybe we should go back to the previous behavior and add your changes (@lfelipe1501) as an optional switch so we don't break people's themes...

**Comment by nwinkler** [comment]
I've reverted the #1095 change in #1101 - sorry for breaking this. I don't use Ruby, which means that I did not notice breaking this when I merged #1095.

@lfelipe1501 Please provide a new PR that adds your additional information as optional, e.g. through a switch.
